contracts/NFTStakingManager.sol

```diff
@@ -64,9 +76,19 @@ struct ValidationInfoView {
   bytes registrationMessage;
 }
 
+enum DelegatorStatus {
```

Comment:
Can we determine the status just by looking at start/endEpoch?

Reply:
Talked offline, decided to keep delegator status so that we can update end epoch on the initiate call and still track that a delegator is pending removal. 

Don't need validator status because that's tracked by ava in the `ValidatorManager`